Makefile

```diff
@@ -46,15 +46,27 @@ generate-mocks: ## Generate mocks
 	@echo "==> $@"
 	@go run github.com/golang/mock/mockgen -destination internal/directory/auth0/mock_auth0/mock.go github.com/pomerium/pomerium/internal/directory/auth0 RoleManager
 
-.PHONY: build-deps
-build-deps: ## Install build dependencies
+.PHONY: build-lint
+deps-lint: ## Install lint dependencies
+	@echo "==> $@"
+	@$(GO) install github.com/client9/misspell/cmd/misspell@${MISSPELL_VERSION}
+	@$(GO) install github.com/golangci/golangci-lint/cmd/golangci-lint@${GOLANGCI_VERSION}
+
+.PHONY: deps-build
+deps-build: ## Install build dependencies
+	@echo "==> $@"
+	@$(GO) install github.com/open-policy-agent/opa@${OPA_VERSION}
+	@$(GO) install github.com/tetratelabs/getenvoy/cmd/getenvoy@${GETENVOY_VERSION}
+
+.PHONY: deps-release
+deps-release: ## Install release dependencies
 	@echo "==> $@"
-	@cd /tmp; GO111MODULE=on $(GO) get github.com/client9/misspell/cmd/misspell@${MISSPELL_VERSION}
-	@cd /tmp; GO111MODULE=on $(GO) get github.com/golangci/golangci-lint/cmd/golangci-lint@${GOLANGCI_VERSION}
-	@cd /tmp; GO111MODULE=on $(GO) get github.com/open-policy-agent/opa@${OPA_VERSION}
-	@cd /tmp; GO111MODULE=on $(GO) get github.com/tetratelabs/getenvoy/cmd/getenvoy@${GETENVOY_VERSION}
 	@cd /tmp; GO111MODULE=on $(GO) get github.com/goreleaser/goreleaser@${GORELEASER_VERSION}
```

Comment:
go releaser is not compatible with the new module install syntax

```
go install github.com/goreleaser/goreleaser@v0.157.0: github.com/goreleaser/goreleaser@v0.157.0
	The go.mod file for the module providing named packages contains one or
	more replace directives. It must not contain directives that would cause
	it to be interpreted differently than if it were the main module.

```